UPGRADE-4.0.md

```diff
@@ -350,3 +350,11 @@ Yaml
    the `!php/object` tag.
 
  * Duplicate mapping keys lead to a `ParseException`.
+
+SecurityBundle
+--------------
+
+ * The `UserPasswordEncoderCommand` class does not allow `null` as first argument anymore.
+ 
+ * `UserPasswordEncoderCommand` does not extend `ContainerAwareCommand` nor implement 
```

Comment:
Not sure we need to talk about the class. Not implementing the interface implies that the command cannot be an instance of `ContainerAwareCommand` anymore.